components/util.js

```diff
@@ -0,0 +1,7 @@
+export function getAgeFromTime(birthDayTime) {
+  let timeDiff = Date.now() - birthDayTime.getTime();
+
+  // I really hope not this is going to last forever!
+  // One magic number magics numbers everywhere!!!
+  return new Date(timeDiff).getFullYear() - 1970;
```

Comment:
you know. Date objects in javascript has getFullYearUTC :3

Reply:
same with month and day

Reply:
Date object backed by unix time. 
unix time only back to 1970. 
that why I have to minues it. 
even if I use getUTCFullYear() I would still have to do it. 
getFullYear is only the localtime where UTC is the universial time. :b

Reply:
You want UTC or local time ?

Reply:
Can't we use the local time of the phone?

Reply:
Let me know If I should change anything  @Joergenson ?

Reply:
On my machine 
```js
console.log(new Date().getUTCHours());
console.log(new Date().getHours());
```
returns the same thing. 
I thought getHours() returned localtime and not UTC.
We can change to another Library, but I will argue 24 hours of at max doesn't matter in our Application.